iGCS/UserInterface/ViewControllers/WaypointMapBaseController.m

```diff
@@ -143,8 +143,12 @@ - (void) configureRoutePolylines:(CLLocationCoordinate2D*)coords withLength:(NSU
 
 // 1e-5 is very approximately 1m
 #define CLLocationCoordinate2DEqual(x, y) (fabs((x).latitude - (y).latitude) < 1e-5 && fabs((x).longitude - (y).longitude) < 1e-5)
+- (void) replaceMission:(WaypointsHolder*)mission
```

Comment:
Add spacing between top level forms.

Might also want to move the #define, as it is associated with the method below this one (not that either location is ideal)